docs/migrations.md

```diff
@@ -141,6 +141,8 @@ export class PostRefactoringTIMESTAMP implements MigrationInterface {
 
 ## Running and reverting migrations
 
+**`typeorm migration:create` and `typeorm migration:generate` will create `.ts` files. The `migration:run` and `migration:revert` commands only work on `.js` files. Thus the typescript files need to be compiled before running the commands.** Alternatively you can use `ts-node` in conjunction with `typeorm` to run `.ts` migration files. 
```

Comment:
@wederer could you please also add an example of using `ts-node`? It should be somewhere in the issues you mentioned

Reply:
Will do.